### PR TITLE
build(npm): use bun

### PR DESCRIPTION
**Pull request by Rubilmax**

### CODE REVIEW

**Review by julien-devatom — COMMENTED** [review]
I've not tested enough bun to have a concrete opinion on it.

File: test/hardhat/Morpho.spec.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ const random = () => {
 const identifier = (marketParams: MarketParamsStruct) => {
   const encodedMarket = AbiCoder.defaultAbiCoder().encode(
     ["address", "address", "address", "address", "uint256"],
-    Object.values(marketParams),
+    Object.values(marketParams)
```
Comment (Rubilmax) [inline test/hardhat/Morpho.spec.ts:25]: I had to downgrade prettier I don't know yet why

**Comment by MathisGD** [comment]
I'm not sure that we should switch to bun, especially in this repo. The performance improvement is cool but we're not sure about the stability and Morpho Labs' devs are much more used to yarn

**Comment by Rubilmax** [comment]
> Morpho Labs' devs are much more used to yarn

If we don't switch to `bun` Morpho Labs devs will never be used to it :D

**Comment by MathisGD** [comment]
Yes good remark, but we could do it elsewhere ^^

**Pull request closed without merging**